config/database.yml

```diff
@@ -21,7 +21,14 @@ default: &default
 
 development:
   <<: *default
+  adapter: postgresql
+  encoding: unicode
   database: backend_kc_development
+  pool: 5
+  username: <%= ENV["BACKEND_DATABASE_USERNAME"] %>
+  password: <%= ENV["BACKEND_DATABASE_PASSWORD"] %>
```

Comment:
If you are adding this create a .env.sample file too and add that too with sample Keys and values.

Reply:
Hi added `.env.sample` file with a random credentials.